gradle.properties

```diff
@@ -1,8 +1,8 @@
-version=7.1.0
-iexecCommonVersion=6.0.1-NEXT-SNAPSHOT
-iexecBlockchainAdapterVersion=7.1.1
-iexecResultVersion=7.1.0
-iexecSmsVersion=7.1.0
+version=gramine
+iexecCommonVersion=gramine-NEXT-SNAPSHOT
+iexecBlockchainAdapterVersion=gramine-NEXT-SNAPSHOT
+iexecResultVersion=gramine-NEXT-SNAPSHOT
+iexecSmsVersion=gramine-NEXT-SNAPSHOT
```

Comment:
do not forget to revert this :pray: and use correct `-NEXT-SNAPSHOT` dependency versions